docs/content/integrations/airlift/tutorial.mdx

```diff
@@ -1 +1,1258 @@
-Empty for now, referenced in reference.mdx.
\ No newline at end of file
+## Example: Migrating an Airflow DAG to Dagster
+
+This example demonstrates how to migrate an Airflow DAG to Dagster using the `dagster-airlift` package. It contains code examples of how to peer, observe, and migrate assets from an Airflow DAG to Dagster. The below guide will walk through each of these steps in detail.
+
+## Example Structure
+
+```plaintext
+tutorial_example
+├── shared: Contains shared Python & SQL code used Airflow and proxied Dagster code
+│
+├── dagster_defs: Contains Dagster definitions
+│   ├── stages: Contains reference implementations of each stage of the migration process
+│   ├── definitions.py: Empty starter file for following along with the tutorial
+│
+├── airflow_dags: Contains the Airflow DAG and associated files
+│   ├── proxied_state: Contains migration state files for each DAG, see migration step below
+│   ├── dags.py: The Airflow DAG definition
+```
+
+## Guide
+
+This tutorial will walk through the process of peering, observing, and migrating assets from an Airflow DAG to Dagster.
+
+First, clone the tutorial example repo locally, and enter the repo directory.
+
+```bash
+gh repo clone dagster-io/airlift-tutorial
```

Comment:
I think I would opt for a `git clone` as it doesn't require `gh` to be installed, unless that's how we're doing it elsewhere.

Reply:
Agreed, but I'll make that change in a follow up.